### PR TITLE
New toolbar style

### PR DESCRIPTION
**Pull request by zbettenbuk**
image

### CODE REVIEW

**Comment by zbettenbuk** [comment]
Updated the black side, I felt it too dark

image

File: react/features/conference/components/native/styles.js [hunk 1]
```diff
@@ -40,11 +41,11 @@ export default {
...
 
     gradientStretchBottom: {
-        height: GRADIENT_HEIGHT + FILMSTRIP_SIZE
+        height: BOTTOM_GRADIENT_HEIGHT
     },
 
-    gradientStretch: {
-        height: GRADIENT_HEIGHT
+    gradientStretchTop: {
+        height: DEFAULT_GRADIENT_SIZE
```
Comment (saghul) [inline react/features/conference/components/native/styles.js]: why default and not top? (name)

Comment (zbettenbuk) [reply react/features/conference/components/native/styles.js]: Because that's the ```minHeight``` of the bottom gradient as well

Comment (saghul) [reply react/features/conference/components/native/styles.js]: Ah, okidoki!

**Pull request merged**